server/src/core/auth/services/auth.service.ts

```diff
@@ -194,8 +194,11 @@ export class AuthService {
   }
 
   async impersonate(userId: string) {
-    const user = await this.userRepository.findOneBy({
-      id: userId,
+    const user = await this.userRepository.findOne({
```

Comment:
@Weiko can we also check that the workspaceMember is allowing impersonation and (I think it's the case), that the user who wants to impersonate has the "canImpersonate" boolean on their user